elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -380,7 +373,7 @@ protected boolean updateToManyRelation(String fieldName, Set<PersistentResource>
         deleted
                 .stream()
                 .forEach(toDelete -> {
-                    checkFieldAwarePermissions(UpdatePermission.class, fieldName);
+                    checkFieldAwarePermissions(UpdatePermission.class, fieldName, null, toDelete);
```

Comment:
I feel like this check is unnecessary and should be removed.  We already check the field permission in the calling function.